js/main.js

```diff
@@ -462,13 +462,18 @@ function fixMapHeight() {
 $(window).on('resize', fixMapHeight);
 
 
-$(window).on('hashchange',function() {
-    // Don't create a new history state, because the hash change already did
-    setCity(getPathCity(), false);
-});
-
+function updateOldUrlScheme(){
```

Comment:
You can shorten the documentation when the function name describes what happens a bit more in detail such as `removeHashPrefixInUrl()` or `removeHashPrefixInPathNamePart()` or similar.